xml/templates/message_templates/case_activity_html.tpl

```diff
@@ -73,27 +73,29 @@
       </tr>
      {/foreach}
 
-     {foreach from=$activity.customGroups key=customGroupName item=customGroup}
-      <tr>
-       <th {$headerStyle}>
-        {$customGroupName}
-       </th>
-      </tr>
-      {foreach from=$customGroup item=field}
-       <tr>
-        <td {$labelStyle}>
-         {$field.label}
-        </td>
-        <td {$valueStyle}>
-         {if $field.type eq 'Date'}
-          {$field.value|crmDate:$config->dateformatDatetime}
-         {else}
-          {$field.value}
-         {/if}
-        </td>
-       </tr>
-      {/foreach}
-     {/foreach}
+     {if !empty($activity.customGroups)}
+       {foreach from=$activity.customGroups key=customGroupName item=customGroup}
+        <tr>
+         <th {$headerStyle}>
+          {$customGroupName}
+         </th>
+        </tr>
+        {foreach from=$customGroup item=field}
+         <tr>
+          <td {$labelStyle}>
+           {$field.label}
+          </td>
+          <td {$valueStyle}>
+           {if $field.type eq 'Date'}
+            {$field.value|crmDate:$config->dateformatDatetime}
+           {else}
+            {$field.value}
+           {/if}
+          </td>
+         </tr>
+        {/foreach}
+       {/foreach}
+      {/if}
```

Comment:
This alignment looks a bit wonky - I guess give the state our tpls that's non-blocking